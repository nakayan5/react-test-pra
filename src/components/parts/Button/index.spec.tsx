import { render } from "@testing-library/react";
import { Button } from "./index";

describe("Button", () => {
  test("render", () => {
    const { asFragment } = render(<Button>ボタン</Button>);
    expect(asFragment()).toMatchSnapshot();
  });
});
