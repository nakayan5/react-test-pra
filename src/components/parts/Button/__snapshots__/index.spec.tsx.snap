// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button render 1`] = `
<DocumentFragment>
  <button
    class="sc-bczRLJ"
  >
    ボタン
  </button>
</DocumentFragment>
`;
